### PR TITLE
git-reflog: update Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/git-reflog.md [hunk 1]
```diff
@@ -13,4 +13,4 @@
 
 - Muestra solo las últimas 5 entradas en el registro de referencias:
 
-`git reflog -n {{5}}`
+`git reflog {{-n|--dry-run}} {{5}}`
```
Comment (nelsonfigueroa) [inline pages.es/common/git-reflog.md:16]: I don't think this is correct. `git reflog -n 5` works, but not `git reflog --dry-run 5`. According to [the docs](https://git-scm.com/docs/git-reflog) the `--dry-run` option is only available for subcommands like `git reflog expire` and `git reflog delete`. 

For reference I tested this out on Git v.2.46.2

**Review by nelsonfigueroa — CHANGES_REQUESTED** [review]

File: pages.es/common/git-reflog.md (hunk 1 above)
Comment (nelsonfigueroa) [reply pages.es/common/git-reflog.md:16]: I'll create a separate PR for pages/common/git-reflog.md to change the example to use `{{-n|--max-count}}`

Comment (nelsonfigueroa) [reply pages.es/common/git-reflog.md:16]: PR created: https://github.com/tldr-pages/tldr/pull/13837

**Review by nelsonfigueroa — APPROVED** [review]

**Review by reinhart1010 — APPROVED** [review]
LGTM

**Pull request merged**